arch/arm/core/aarch64/reset.S

```diff
@@ -63,7 +63,7 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 	isb
 
 	/* Initialize sctlr_el3 to reset value */
-	mov_imm	x1, SCTLR_EL3_RES1
+	mov_imm	x1, (SCTLR_EL3_RES1 | SCTLR_EIS | SCTLR_EOS)
```

Comment:
The problem I have with this is that `SCTLR_EL3_RES1` is still a magic value to me. Can you point me to where the bits enabled in there are defined? Because in the reference manual I have bits 29, 28, 23, 18, 16, 5 and 4 are `reserved`.

Reply:
About `EIS` and `EOS` since those are for `v8.5` I indeed moved there in `plat_el3_init`

Reply:
Again, I see now that you are initializing in one step all the `RES1` bits for `v8.0`